src/styles/normalize.css

```diff
@@ -0,0 +1,4 @@
+* {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
```

Comment:
```suggestion
 
```
you can delete this propety
font-weight: **400**; is used by default for all font-families